### DISCUSSION TITLE
默认播放器模式下的两个开关按钮开启任一个后无法正常切换模式

### DISCUSSION THREAD

**Discussion by shuaiyang2000** [the1812/Bilibili-Evolved#2537] [提问 / Q&A] [answered]
1.默认播放器模式下的两个开关按钮开启任一个后无法正常切换模式，关闭后可正常切换；但设置为宽屏时，播放器会自动向下偏移一段距离（没有设置定位偏移量；关闭脚本后也会向下偏移）；
2.播放时自动关灯一直无效；

**Comment by Geeyun-JY3** [comment, marked as answer]
那就是指 v1

宽屏估计还是：#483
自动关灯：#2399

**Reply by Geeyun-JY3** [reply]
你要不先试试 v2 还有没有 1 的问题

**Reply by shuaiyang2000** [reply]
v2可以解决；关灯没问题，但并不是每一个视频都可以宽屏
